index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: CockroachDB Docs
+summary: CockroachDB is a distributed SQL database built on a transactional and strongly-consistent key-value store.
```

Comment:
"Read the documentation for CockroachDB, a distributed SQL database built on.... 

[This is to distinguish this meta description from the main homepage's meta description.]